schemas/com.snowplowanalytics.snowplow.media/ad_break/jsonschema/1-0-0

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity, shared with all media_player_ad events belonging to the ad break.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "ad_break",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "Ad break name (e.g., pre-roll, mid-roll, and post-roll).",
+      "type": [
+        "null",
+        "string"
+      ],
+      "maxLength": 4096
+    },
+    "breakId": {
+      "type": "string",
+      "maxLength": 256,
+      "description": "An identifier for the ad break."
+    },
+    "startTime": {
+      "type": "number",
+      "description": "Playback time in seconds at the start of the ad break.",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "breakType": {
+      "description": "Type of ads within the break: linear (take full control of the video for a period of time), nonlinear (run concurrently to the video), companion (accompany the video but placed outside the player).",
+      "enum": [
+        "linear",
+        "nonlinear",
+        "companion",
+        null
+      ],
+      "type": [
+        "string",
+        "null"
+      ]
```

Comment:
Should there be like an optional `podSize` here to complement `podPosition` in `ad`?

Reply:
I think that is useful! Have added the `podSize` property to ad break schema.